### PR TITLE
doc: Update release notes changelog

### PR DESCRIPTION
**Pull request by MarekPieta**
Change includes updates for: Profiler, Common Application Framework (CAF), Pelion, Edge Impulse and nRF Desktop.

Jira: NCSDK-11010

### CODE REVIEW

**Review by rakons — APPROVED** [review]

**Review by pdunaj — APPROVED** [review]

File: doc/nrf/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -98,6 +98,15 @@ nRF Desktop
...
 * Settings backend changed from FCB to NVS.
+* Switched to using :ref:`caf_power_manager`.
+* Added clearing button state reported over BLE if USB was connected while button was pressed.
+  This fixed issue related to reporting wrong button state over BLE.
+* Added support for HID keyboard LEDs output report.
+  The report is handled by the nRF Desktop peripherals and forwarded by the nRF Desktop dongles.
+* Added nRF5340DK working as a nRF Desktop dongle.
+* Added forwarding HID boot reports in :ref:`nrf_desktop_hid_forward`.
+* Fixed generating motion in :ref:`nrf_desktop_motion` (``motion_buttons`` and ``motion_simulated``) while HID boot protocol is in use.
+* Added GPIO LEDs to nrf52820dongle.
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:109]:
```suggestion

Updated:

* Changed settings backend from FCB to NVS.
* Switched to using :ref:`caf_power_manager`.
* Fixed an issue with generating motion in :ref:`nrf_desktop_motion` (``motion_buttons`` and ``motion_simulated``) while the HID boot protocol was in use.

Added:

* Added a functionality to clear the button state reported over Bluetooth LE if the USB was connected while the button was pressed.
  This also fixes an issue related to reporting wrong button state over Bluetooth LE.
* Added support for HID keyboard LED output report.
  The report is handled by the nRF Desktop peripherals and forwarded by the nRF Desktop dongles.
* Added support for nRF5340 DK working as an nRF Desktop dongle.
* Added a functionality for forwarding HID boot reports in :ref:`nrf_desktop_hid_forward`.
* Added GPIO LEDs to the ``nrf52820dongle`` board.

File: doc/nrf/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -166,10 +175,29 @@ Modem library
...
+Profiler
+--------
+
+* Added profiling string data.
+* Optimized numeric data encoding.
+
+Edge Impulse
+------------
+
+* Added Thingy:53 support to :ref:`nrf_machine_learning_app`.
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:193]:
```suggestion
* Added support for Thingy:53 to :ref:`nrf_machine_learning_app`.
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 3]
```diff
@@ -166,10 +175,29 @@ Modem library
...
+* Added nRF52840DK configuration with data forwarder over NUS to :ref:`nrf_machine_learning_app`.
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:194]:
```suggestion
* Added configuration for nRF52840 DK that supports data forwarder over NUS to :ref:`nrf_machine_learning_app`.
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 4]
```diff
@@ -166,10 +175,29 @@ Modem library
...
+
 Pelion
 ------
 
 * Updated Pelion Device Management Client library version to 4.10.0.
+* Switched to using :ref:`caf_power_manager` and :ref:`caf_net_state`.
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:200]: Isn't this valid only for the Pelion Client app?

**Review by greg-fer — CHANGES_REQUESTED** [review]
Couple of edits.

File: doc/nrf/releases/release-notes-changelog.rst (hunk 1 above)
Comment (MarekPieta) [reply doc/nrf/releases/release-notes-changelog.rst:109]: `This fixes an issue related to reporting wrong button state over Bluetooth LE`
This is the bug that was fixed by this change. No need to use `also` here.

Comment (MarekPieta) [reply doc/nrf/releases/release-notes-changelog.rst:109]: `nrf52820dongle_nrf52820` ?
That's complete board name used while building application

File: doc/nrf/releases/release-notes-changelog.rst (hunk 4 above)
Comment (MarekPieta) [reply doc/nrf/releases/release-notes-changelog.rst:200]: Yes, updated.

**Review by greg-fer — APPROVED** [review]

**Pull request merged**